### ISSUE TITLE
Implement Debug Inspection

### ISSUE THREAD

**Issue by ChrisCoffey** [ensime/ensime-vim#213]
We have not implemented the Ensime debug inspection api. Like stepping, you can work around this with other tools but it's probably simplest to implement the API directly.

**Comment by 2chilled** [comment]
@ChrisCoffey I'm not clear about the workarounds for missing debug inspection, can you give me a hint?

**Comment by ChrisCoffey** [comment]
Oh, I was just talking about how it's possible to set manual breakpoints in lieu of stepping. Not a true workaround. I've updated the description to remove the ambiguity.

**Comment by ChrisCoffey** [comment]
updated to remove the ambiguity

**Comment by jvican** [comment]
Yes, I definitely agree we should implement this API and not rely on other third parties.
